hbase-client/src/main/java/org/apache/hadoop/hbase/client/RpcRetryingCallerImpl.java

```diff
@@ -201,10 +221,20 @@ private long singleCallDuration(final long expectedSleep) {
   public T callWithoutRetries(RetryingCallable<T> callable, int callTimeout)
     throws IOException, RuntimeException {
     // The code of this method should be shared with withRetries.
+    long startTime = EnvironmentEdgeManager.currentTime();
```

Comment:
```suggestion
```
nit; Left over from some testing? I notice it's in the docs for `OperationInterceptor` as well